src/apps/reservation-list/list/reservation-list-pagination.test.ts

```diff
@@ -373,31 +373,32 @@ describe("Reservation list pagination", () => {
     ]);
 
     // ID 11 2.b.iv more than "ready for pickup" 10 reservations the items paginate (here 2, because of config in cy.visit)
-    cy.get(".list-reservation-container")
+    cy.get("[data-cy='list-reservation-container']")
```

Comment:
Please use [cy.getBySel](https://github.com/danskernesdigitalebibliotek/dpl-react/blob/1bcccbc610b229fbe1dbe97c33d4624396e653b5/docs/code_guidelines.md#test-structuring)
 everywhere in this file ;)